### ISSUE TITLE
Handle window undefined error server side

### ISSUE THREAD

**Issue by nirav-panchal** [htdangkhoa/react-ssr-starter#346]
How to handle ReferenceError: window is not defined on server side while client side libraries load which used window object?

**Comment by nsit8919** [comment]
@nirav-panchal  is SSR working in example ?

**Comment by nirav-panchal** [comment]
@nsit8919 @htdangkhoa Yes, Html is not loading on server side, even code splitting is not working too.

**Comment by nsit8919** [comment]
any suggestions?

**Comment by nsit8919** [comment]
is it working or not  ? any suggestion how it's works

**Comment by nsit8919** [comment]
do you have any other boilerplate >
